package.json

```diff
@@ -4,27 +4,46 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
```

Comment:
Your main file is index.js but you have implemented an app.js how come?

Reply:
I apologize.
I did not want to tamper with the existing setup so I created a seperate folder called "server" for the sequelize conversion, that was why I created a new file